src/components/OcCheckbox.spec.js

```diff
@@ -0,0 +1,67 @@
+import OcCheckbox from "./OcCheckbox.vue"
+import { mount } from "@vue/test-utils"
+
+describe("OcCheckbox", () => {
+  function getWrapperWithProps(props) {
+    return mount(OcCheckbox, {
+      propsData: {
+        label: "test label",
```

Comment:
may be this can be removed or ?
because you are still using as `const wrapper = getWrapperWithProps({ id: "test-id", label: "test label" })` (`L:25` )

Reply:
label is a required prop, so it is set as default value for wrapper getter

Reply:
if so then you can remove `label` from here
https://github.com/owncloud/owncloud-design-system/blob/2236c0d897588315842ffa14601804b9cd996656/src/components/OcCheckbox.spec.js#L25